### PR TITLE
Hotfix for #856 - switch to using ephemeral JES pipelines

### PR DESCRIPTION
**Pull request by geoffjentry**
@mcovarr @Horneth 

I haven't looked but I'm guessing the non-hotfix will be nearly (if not fully) identical so I'll wait until this is done before proceeding on the real one.

### CODE REVIEW

File: engine/src/main/scala/cromwell/engine/backend/jes/Run.scala [hunk 1]
```diff
@@ -25,48 +27,60 @@ object Run  {
...
       runId
     }
 
-    // Only run the pipeline if the pipeline ID is defined.  The pipeline ID not being defined corresponds to a
-    // resumption of a previous run, and runIdForResumption will be defined.  The Run code takes care of polling
-    // in both the newly created and resumed scenarios.
-    val runId = if (pipeline.pipelineId.isDefined) runPipeline else pipeline.runIdForResumption.get
-    new Run(runId, pipeline, logger)
+    // If runIdForResumption is defined use that, otherwise we'll create a new Run with an ephemeral pipeline.
+    // The Run code takes care of polling.
```
Comment (mcovarr) [inline engine/src/main/scala/cromwell/engine/backend/jes/Run.scala]: I realize this comment was refactored from before but I still don't know what it's trying to say.  😦

**Comment by mcovarr** [comment]
:+1:

File: engine/src/main/scala/cromwell/engine/backend/jes/Run.scala (hunk 1 above)
Comment (geoffjentry) [inline engine/src/main/scala/cromwell/engine/backend/jes/Run.scala]: I'm too lazy to look at blame mode but I thought you wrote it :)

**Comment by mcovarr** [comment]
:+1: reinstated

[Approved with PullApprove](https://pullapprove.com/broadinstitute/cromwell/pull-request/860/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/cromwell)

**Comment by Horneth** [comment]
👍

**Pull request merged**